scroll-animations-1/Overview.bs

```diff
@@ -385,12 +383,14 @@ interface ScrollTimeline : AnimationTimeline {
   readonly attribute ScrollDirection orientation;
   readonly attribute DOMString startScrollOffset;
   readonly attribute DOMString endScrollOffset;
-  readonly attribute (double or ScrollTimelineAutoKeyword) timeRange;
 };
 </pre>
 
 A <dfn>scroll timeline</dfn> is an {{AnimationTimeline}} whose time values are determined
 not by wall-clock time, but by the progress of scrolling in a [=scroll container=].
+The {{AnimationTimeline/duration}} of a <a>scroll timeline</a> is 100s to provide
```

Comment:
It might be nice if this were still a percentage but the definition of [time values](https://www.w3.org/TR/web-animations-1/#time-value-section) already suggests the correspondence to milliseconds is weak.